modules/op-monitoring-pipeline-run-status-using-pipelines-as-code.adoc

```diff
@@ -25,11 +25,11 @@ When {pac} detects an error in one of the tasks of a pipeline, a small snippet c
 [discrete]
 .Annotations for log error snippets
 
-In the {pac} config map, if you set the `error-detection-from-container-logs` parameter to `true`, {pac} detects the errors from the container logs and adds them as annotations on the pull request where the error occurred.
+In the `TektonConfig` custom resource, under `pipelinesAsCode` and `settings`, if you set the `error-detection-from-container-logs` parameter to `true`, {pac} detects the errors from the container logs and adds them as annotations on the pull request where the error occurred.
 
 [IMPORTANT]
 ====
-This feature is in Technology Preview. 
+This feature is in Technology Preview.
 ====
```

Comment:
There is specific guidance for adding a notice about tech preview features: see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features